domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/validation/eventcasefieldcomplextype/EventComplexTypeShowConditionValidatorImpl.java

```diff
@@ -50,7 +53,7 @@ public ValidationResult validate(EventComplexTypeEntity eventCaseFieldEntity,
         List<EventCaseFieldEntity> allEventCaseFieldEntitiesForEventCase =
             eventCaseFieldEntityValidationContext.getAllEventCaseFieldEntitiesForEventCase();
 
-        List<String> allSubTypePossibilities = CaseFieldEntityUtil.buildDottedComplexFieldPossibilities(
```

Comment:
and this block of code is very similar to the one in EventCaseFieldShowConditionValidatorImpl. I think `FieldsPathValidator` class is a must now. Too much duplication